lexer/src/main/kotlin/lexer/TokenMap.kt

```diff
@@ -2,14 +2,15 @@ package lexer
 
 import token.TokenMatcher
 import token.TokenType
+import version.Version
 import java.util.EnumMap
 
-fun getTokenMap(): EnumMap<TokenType, TokenMatcher> {
+fun getTokenMap(versionNumber: Version = Version.V1): EnumMap<TokenType, TokenMatcher> {
     val tokenMap: EnumMap<TokenType, TokenMatcher> = EnumMap(TokenType::class.java)
 
     // KEYWORDS
-    tokenMap[TokenType.LET] = TokenMatcher(TokenType.LET, "let")
-    tokenMap[TokenType.PRINTLN] = TokenMatcher(TokenType.PRINTLN, "println")
+    tokenMap[TokenType.LET] = TokenMatcher(TokenType.LET, "\\blet\\b")
+    tokenMap[TokenType.PRINTLN] = TokenMatcher(TokenType.PRINTLN, "\\bprintln\\b")
```

Comment:
I think this regex should not have the \b at the end, since we want to do something like `prinln()`